packages/web-react/src/types/spinner.ts

```diff
@@ -0,0 +1,8 @@
+import { StyleProps, TextColorsDictionaryType, TransferProps } from './shared';
+
+export interface SpinnerProps extends StyleProps, TransferProps {}
+
+export interface SpiritSpinnerProps<T = void> extends SpinnerProps {
+  /** Color of the Spinner */
+  color?: TextColorsDictionaryType<T>;
```

Comment:
I will prefer to use `C` for Color.

```suggestion
  color?: TextColorsDictionaryType<C>;
```